repo/packages/O/openvpn/1/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "packagingVersion": "2.0",
+  "name": "openvpn",
+  "version": "0.0.0-0.2",
+  "scm": "https://github.com/dcos-labs/dcos-openvpn",
+  "description": "OpenVPN server and REST admin interface - community package",
+  "maintainer": "https://dcos.io/community/",
+  "tags": ["vpn", "remote-access"],
+  "preInstallNotes": "This release supercedes and merges both the openvpn and openvpn-admin 0.0.0-0.1 packages.\nPlease remove the openvpn-admin package if installed.\nNew features include basic auth, TLS and Zookeeper znode ACL.\n Please ensure you configure the ovpn_username and ovpn_password environment variables before deployment,\nas these are essential for access control to the REST interface and Zookeeper.\nPlease read the installation notes on https://github.com/dcos-labs/dcos-openvpn.",
```

Comment:
Add this line at the beginning of preInstallNotes: `Community packages are unverified and unreviewed content from the community`

Reply:
Done